iceoryx_utils/test/moduletests/test_posix_mutex.cpp

```diff
@@ -80,15 +80,15 @@ TEST_F(Mutex_test, DestructorFailsOnLockedMutex)
             std::thread* t;
             {
                 iox::posix::mutex mtx{false};
-                iox::posix::Timer hold(1000_ms);
+                iox::clock::DeadlineTimer hold(1000_ms);
```

Comment:
Naming the timer ``hold`` seems weird. Why not ``timer``? (I understand it was like this before but still, we can try to improve these little things)

Reply:
Yes. modified to mutexTimer.